app/elm/Creature.elm

```diff
@@ -49,24 +50,29 @@ health creature =
         (toFloat creature.hitPoints / toFloat (maxHp creature.creatureType)) * 100
 
 
-attack : Creature -> Creature -> Attack
-attack attacker victim =
+attack : Creature -> Creature -> Seed -> ( Attack, Seed )
+attack attacker victim seed =
     let
-        damage =
+        damageGenerator =
             case attacker.creatureType of
                 Wizard ->
-                    4
+                    int 0 4
```

Comment:
This is "hackathon fine", but this code would probably be clearer with the namespaced version of this function: `Random.int`